composer.json

```diff
@@ -26,6 +26,7 @@
         "drupal/devel": "dev-8.x-1.x",
         "drupal/drupal-extension": "dev-assert-not-button as 3.1.6",
         "drupal/phing-drush-task": "1.0",
+        "drupal/search_api": "8.1.0-alpha12",
```

Comment:
This hardcodes the alpha12 release. If you put a tilde in front of the version string, then we will also benefit from updates when they are released: `~8.1.0-alpha12`.

This should also be moved to the "require" section. It is now in "require-dev" which is intended for development environments only. Search API is something we need on acceptance, staging and production as well.